tech/languages/python/python-installation.md

```diff
@@ -8,6 +8,28 @@ version: 2.7.10
 
 # Python Installation
 
-Python is installed by default in Fedora.
+## CPython
 
-Currently we have two releases; Python-2.7.10 and Python-3.4.2
+To install Python 2, simply type:
+
+```
+$ sudo dnf install python
+```
+
+Similarly for Python 3:
+
+```
+$ sudo dnf install python3
+```
+
+The above command will install the latest stable Python 2/3 package and also the useful python-pip (more on that one in another article).
```

Comment:
We should not say 'more on that somewhere'. If we have the content somewhere it should be linked (from the `python-pip`.)